index.html

```diff
@@ -0,0 +1,1209 @@
+
+                  <p>Clients can discover the ACL resource associated with a resource by making an HTTP request on the target URL, and checking the HTTP <code>Link</code> header with the <code>rel</code> parameter [<cite><a class="bibref" href="#bib-rfc8288">RFC8288</a></cite>].</p>
+
+                  <div class="note" id="acl-resource-uri-security" inlist="" rel="schema:hasPart" resource="#uri-security">
+                    <h4 property="schema:name"><span>Note</span>: ACL Resource URI Security</h4>
+                    <div datatype="rdf:HTML" property="schema:description">
+                      <p>The URI of an ACL resource does not in itself pose a security threat ([<cite><a class="bibref" href="#bib-rfc3986">RFC3986</a></cite>] security considerations). This specification does not constrain the discoverability of ACL resources.</p>
+                    </div>
+                  </div>
+
+                  <div class="note" id="uri-origin" inlist="" rel="schema:hasPart" resource="#uri-origin">
+                    <h4 property="schema:name"><span>Note</span>: URI Origin</h4>
+                    <div datatype="rdf:HTML" property="schema:description">
+                      <p>The resource and the associated ACL resource can be on different origins [<cite><a class="bibref" href="#bib-rfc6454">RFC6454</a></cite>].</p>
+                    </div>
+                  </div>
+
+                  <p>Clients MUST NOT derive the URI of the ACL resource through string operations on the URI of the resource.</p>
+
+                  <p class="advisement">Source: <a href="https://github.com/solid/web-access-control-spec/issues/8" rel="cito:citesAsSourceDocument">issues/8</a>, <a href="https://github.com/solid/web-access-control-spec/issues/62" rel="cito:citesAsSourceDocument">issues/62</a>, <a href="https://github.com/solid/specification/issues/131" rel="cito:citesAsSourceDocument">issues/131</a>, <a href="https://github.com/solid/specification/issues/176" rel="cito:citesAsSourceDocument">issues/176</a></p>
+                </div>
+              </section>
+
+              <section id="acl-resource-representation" inlist="" rel="schema:hasPart" resource="#acl-resource-representation">
+                <h3 property="schema:name">ACL Resource Representation</h3>
+                <div datatype="rdf:HTML" property="schema:description">
+                  <p>An ACL resource is an <em>RDF document</em> [<cite><a class="bibref" href="#bib-rdf11-concepts">RDF11-CONCEPTS</a></cite>] that can hold any information, typically comprises an unordered set of <a href="#authorization">Authorizations</a>, any of which could permit an attempted access.</p>
+
+                  <p>Servers MUST accept an HTTP <code>GET</code> request targeting an ACL resource when the value of the <code>Accept</code> header requests a representation in <code>text/turtle</code> [<cite><a class="bibref" href="#bib-turtle">TURTLE</a></cite>].</p>
```

Comment:
IIUC, for Solid, this is covered by Solid's requirement to serve both turtle and json-ld, right?

Reply:
If implementers read this here while implementing Solid-including-WAC, they may incorrectly interpret it as only turtle is required, so maybe point to https://github.com/solid/specification/blob/main/protocol.html#L691

Reply:
Correct.